mintapi/api.py

```diff
@@ -171,7 +175,7 @@ def get_session_cookies(self, username, password):
         try:
             from selenium import webdriver
             driver = webdriver.Chrome()
-        except Exception as e:
+        except MintException as e:
```

Comment:
I don't believe this should be MintException. Other python exceptions (ImportError among others) could be raised and we want to catch them all!